src/terraform/install.sh

```diff
@@ -28,9 +28,10 @@ TERRAFORM_DOCS_SHA256="${TERRAFORM_DOCS_SHA256:-"automatic"}"
 
 TERRAFORM_GPG_KEY="72D7468F"
 TFLINT_GPG_KEY_URI="https://raw.githubusercontent.com/terraform-linters/tflint/master/8CE69160EB3F2FE9.key"
-GPG_KEY_SERVERS="keyserver hkp://keyserver.ubuntu.com
+GPG_KEY_SERVERS="keyserver hkps://keyserver.ubuntu.com
 keyserver hkps://keys.openpgp.org
-keyserver hkp://keyserver.pgp.com"
+keyserver hkps://keyserver.pgp.com"
+KEYSERVER_PROXY=${HTTPPROXY:-$HTTP_PROXY}
```

Comment:
Changed the environment variable HTTP_PROXY to the default if HTTPPROXY is not specified in the Feature option.

Reply:
This is the result of testing.

* Test using devcontainers features test in internet direct environment .. OK
* Manually start the container and run install.sh .. OK

Could you please review this fix?